src/main/java/com/github/bn1knb/newspaper/auth/service/SecurityServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.github.bn1knb.newspaper.auth.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.stereotype.Service;
+
+@Service
+
```

Comment:
There shouldn't be Security in this branch. It's only about jpa.
And remove unnecessary space between annotation and class name.